backends/qualcomm/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ if(${ANDROID})
     find_library(android_log log)
 endif()
 
-add_compile_options("-Wall" "-Werror" "-Wno-sign-compare")
+add_compile_options("-Wall" "-Wno-sign-compare")
```

Comment:
May I know why deprecate `-Werror`?

Reply:
I suggest not treating warnings as errors since it may interrupt our release build flow. Currently, the clang++ compiler indicates that the '-s' option is unused. Would you recommend removing this option from CMakeLists.txt to prevent such issues?

Reply:
I see, could you append `"-Wno-unused-command-line-argument"` to the `add_compile_options` in line57?

Reply:
Absolutely!

Reply:
Please add back `-Werror`, thanks.

Reply:
Sure! I added it back~